### DISCUSSION TITLE
Do we need a QR Generator or not?

### DISCUSSION THREAD

**Discussion by mohammadKarimi** [mohammadKarimi/SwiftLink#78] [Ideas]
we can generate QR code for each link, is it valuable or not ? please feel free to discuses.

**Comment by Kamyab7** [comment]
> we can generate QR code for each link, is it valuable or not ? please feel free to discuses.

Absolutely worth it.

**Reply by mohammadKarimi** [reply]
I have benchmarked several excellent services, such as Bitly. Almost all of them provide a QR code generator as well as an attribute for creating shortlinks.

**Reply by Kamyab7** [reply]
Yeah, almost all URL shortener services provide this feature.
